app.rb

```diff
@@ -0,0 +1,133 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def start_console
+    puts 'Welcome to my School Library!'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thank You for using my School Library!'
+        break
+      end
+
+      option input
+    end
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each { |person| puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}" }
+  end
```

Comment:
- [ ] Good job so far 👍 when I create a teacher and select the option to list all persons, the name does not show. Kindly fix this issue 
- [ ] The same issue occurs after creating a rental and listing the rentals. The name does not show. It would be great if you could address this issue too 😄 
- Your console:
![Annotation 2022-04-08 093753](https://user-images.githubusercontent.com/85113567/162399228-9abf485d-4284-4be2-b5dc-6c274ff98c9f.jpg)